lua/neo-tree/defaults.lua

```diff
@@ -491,6 +493,61 @@ local config = {
       },
     },
   },
+  document_symbols = {
+    renderers = {
+      root = {
+        {"indent"},
+        {"icon", default="C" },
+        {"name", zindex = 10},
+      },
+      symbol = {
+        {"indent"},
+        {"icon", default="C" },
```

Comment:
I feel like the `kind_icon` belongs here by default, between the expander and the name.